IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MemberTypeCatenaryStay/DocPropertySet.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MemberTypeCatenaryStay" UniqueId="b15eff21-bd3f-df08-8b2a-e9722e73b14d" ApplicableType="IfcMember/BRACE" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Applicatable type does not match description is this supposed to be BRACE or STAY_CABLE?

Reply:
You're right STAY_CABLE is better/ Commit : 15b79d25de7519c2d0bf4c1d8a3f413d95e3ee63